Signal/src/call/UserInterface/CallUIAdapter.swift

```diff
@@ -63,6 +63,7 @@ extension CallUIAdaptee {
 
         guard self.callService.call == nil else {
             Logger.info("unexpectedly found an existing call when trying to start outgoing call: \(recipientId)")
+            //TODO terminate existing call.
```

Comment:
I think that's a great idea, but this fits into the larger pattern where CallService often ends up with a stale call.  +1 to the proposed fix, but that won't help us where incoming calls are rejected due to stale calls.  I'd really like to look at the root cause(s) of this issue.